overcast-sonos.py

```diff
@@ -95,15 +101,15 @@ def getMetadata(id, index, count):
                         'albumArtist': episode['podcast_title'],
                         'albumArtURI': episode['albumArtURI'],
                         'genreId': 'podcast',
-                        'duration': episode['duration'],
                         'canResume': True,
                     }
                 }
             })
 
     elif id == 'podcasts':
-        podcasts = overcast.get_all_podcasts()
-        response = {'getMetadataResult': [{'index': 0, 'count': len(podcasts), 'total': len(podcasts)}]}
+        all_podcasts = overcast.get_all_podcasts()
+        podcasts = all_podcasts[index:index+count]
+        response = {'getMetadataResult': [{'index': index, 'count': len(podcasts), 'total': len(all_podcasts)}]}
```

Comment:
Paging is sort of useless; Overcast limits it to 250 episodes usually and less than that in some cases. Sonos will only read 100 at a time, so this simply fixes that problem. In the future we could cache this somewhere to avoid the same network call multiple times, but it's not that important.